css/src/tokens/aspect-ratios.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @sass-export-section="aspect-ratios"
+ */
+$aspect-ratios: (
```

Comment:
I'm in favor of keeping this closer to the atomics. Is this a useful token? Does it need sharing between components?

Reply:
Deleted token, moved to atomic style file.